src/rust/grapl-web-ui/src/main.rs

```diff
@@ -57,5 +73,8 @@ async fn main() -> Result<(), GraplUiError> {
     .run()
     .await?;
 
+    // sending remaining spans. Do we need this?
+    global::shutdown_tracer_provider();
+
```

Comment:
Looks like the following might do the trick instead of install_simple
https://github.com/open-telemetry/opentelemetry-rust/blob/main/examples/actix-http-tracing/src/main.rs#L33

Reply:
So the problem is that this is for a significantly earlier version of the library prior to install_simple and install_batch replacing the install method. To fix this I see a few options:
1. Downgrade the library until it works (not a huge fan since we'll want to upgrade it at some point)
2. Use an alternate runtime like async-std
3. Leave it using the simple exporter, which will work for now, but will likely need to be replaced after client 2 or 3.

I'm going to experiment with option 2. I've also found an example of someone creating a library (https://github.com/rthomas/rust-tonic-jaeger-example), with recent versions which I'm going to try to do since that's the endgoal for rust